### ISSUE TITLE
nodal g:model Address generates 'Addres' model, migration

### ISSUE THREAD

**Issue by paulhart** [keithwhor/nodal#164]
The description says it all, but:
```
$ nodal g:model Address
Create: ./app/models/addres.js
Create: ./db/migrations/2016021002484238__create_addres.js
$ 
```
I'll add that I'm aware that `nodal g:model Addresses` does the expected action, creating an Address model. I'm raising this primarily because the screencast gives examples of using singular names for models (i.e. `nodal g:model Tweet`), so I believe there's an expectation that model names should be provided as singular.

**Comment by abossard** [comment]
Still exists in 0.7.3 and I think model names should not be depluralized, but used at is. 
But besides that, this is not an error of nodal per se, but of the inflect library. See here:
https://github.com/pksunkara/inflect/issues/20

**Comment by abossard** [comment]
What about:

    let modelName = inflect.camelize(args[0][0]); 

instead of using classify? Than it would not be singular.

**Comment by keithwhor** [comment]
want to have model names singularized. makes more sense. `new User()` not `new Users()`.

Gonna have to submit a PR on the inflect library.

**Comment by abossard** [comment]
Hi Keith, 

in the documentation here: https://github.com/keithwhor/nodal/blob/master/LEARN.md#models you suggest to use ModelName which implies using singular camemlized names. Why do you call classify on it? Just for the case, that the user would supply a not camelized name?

Anyways, what about just using camelized, which does not have any effect on an already camelized word and emphase in the documentation, that singular names should be used?

I'll quickly wind up a pull request, but feel free to dismiss it.

**Comment by keithwhor** [comment]
Yeah. Just incase the user doesn't do it correctly. I'd rather patch the real issue here, and that's the `inflect` library not performing as expected.

**Comment by abossard** [comment]
I thought about it a bit and I think the user should decide, whether he wants plural or singular name and nodal should use Camelize and Underscore instead of Tableize and Classify.

**Comment by keithwhor** [comment]
@abossard Initially I disagreed, but I'm actually leaning towards dropping the `inflect` package altogether. It produces unwanted side effects and is a bit too magical. I'm going to see what it looks like to remove.
